java/arcs/android/crdt/CrdtEntityReferenceProto.kt

```diff
@@ -0,0 +1,15 @@
+package arcs.android.crdt
+
+import android.os.Parcel
+import arcs.android.util.readProto
+import arcs.core.crdt.CrdtEntity
+
+/** Constructs a [CrdtEntity.ReferenceImpl] from the given [CrdtEntityReferenceProto]. */
+fun fromProto(proto: CrdtEntityReferenceProto) = CrdtEntity.ReferenceImpl(proto.id)
```

Comment:
Can this be an extension function on CrdtEntityReferenceProto?

E.g. `fun CrdtEntityReferenceProto.toReferenceImpl()`

Reply:
Then the call-site below could drop the `let`.

```kotlin
readProto(...)?.toReferenceImpl()
```

Reply:
Good idea, done